contract_layout_category_hide_detail/README.rst

```diff
@@ -0,0 +1,120 @@
+====================================
+Contract layout category hide detail
+====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcontract-lightgray.png?logo=github
+    :target: https://github.com/OCA/contract/tree/12.0/contract_layout_category_hide_detail
+    :alt: OCA/contract
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/contract-12-0/contract-12-0-contract_layout_category_hide_detail
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/110/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to check/uncheck 'Show details' and
+'Show subtotal' flags for *Recurring invoices section lines*
+in Customer Contracts and Supplier Contracts so that
+Invoices are generated with this information.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Invoicing -> Customers -> Customer Contracts* and create
+   a new *Quotation*.
+#. In *Recurring Invoices* tab add a section line.
+#. Add some product lines.
+#. Add another section line, but this time, click on the *plus-circle*
```

Comment:
another? there's no section for now.

Reply:
Two spets before users have to create a secction: 
```In *Recurring Invoices* tab add a section line.```  https://github.com/OCA/contract/pull/489/files#diff-ae6e52733f87c5411e73b799af389834R45